utils/rpms/daos.spec

```
@@ -223,11 +223,7 @@ Requires: libcapstone-devel
 Requires: Lmod
 Requires: capstone-devel
 %endif
-%if (0%{?rhel} >= 8)
-Requires: fuse3-devel >= 3
-%else
-Requires: fuse3-devel >= 3.4.2
-%endif
+Requires: fused >= 1
```

Ditto, if this really should be a change from `fused-devel` to `fused` but it's not clear why the `-devel` is not needed here now.

So either:

```suggestion
```

or

```suggestion
Requires: fused-devel
```